flake.lock

```diff
@@ -0,0 +1,450 @@
+{
```

Comment:
Is this file supposed to be committed, or should it be added to `.gitignore`?  I don't actually know how nix flakes work, it just seemed odd to have this file added in a commit to get svg-local coords.

Reply:
It's not necessary to commit the lock file but doing so pins all the deps used by the flake.  Committing the file will allow you to roll back any breakage cause by a flake update.  You're right, it should have gone in #9 but I neglected to add it and the hie.yaml file, so I put it in here.

Reply:
Ah, ok, makes sense!